server/app.js

```diff
@@ -26,7 +26,7 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
 if (process.env.ASSET_COMPUTE_DEV_TOOL_ENV !== 'development') {
-    app.use(express.static(path.join(__dirname, 'client-build')));
+    app.use(express.static(__dirname + '\\client-build'));
```

Comment:
We need this to work for all OS. We can use the [`process.platform`](https://nodejs.org/api/process.html#process_process_platform) variable to determine if it is Linux, windows, etc.

If Linux or darwin, 
```
app.use(express.static(path.join(__dirname, 'client-build')));
```
if win32,
```
app.use(express.static(__dirname + '\\client-build'));
```